README.md

```diff
@@ -1,9 +1,117 @@
 <a href="https://www.lacework.com/"><img src="https://www.lacework.com/wp-content/uploads/2019/07/Lacework_Logo_color_2019.svg" width="250px" height="100px" title="Lacework" alt="Lacework"></a>
 
-# Lacework Agent Release
-The source for officially supported Lacework agent releases
+# Lacework Agent Releases
+This Lacework agent GitHub site provides the officially supported Lacework agent releases. From this site, you can download and install Lacework agent releases. 
 
-# Lacework Agent Documentation
-Install and configure agent, use the polygraph, and learn about the host dossiers <a href="https://support.lacework.com/hc/en-us/categories/360001044834-Lacework-for-Workload-Security" target="_blank">**here**</a>.
+# Releases
+In the right frame, click **Releases** to view all the available agent releases.
+Under the **Assets**, all the release download files are listed. These download files contain the agent installers. In addition, each release contains a direct link to the specific agent release notes and the ```docker pull``` command for pulling down a Lacework docker install image. 
+For all the agent release notes, see [Release Updates](https://support.lacework.com/hc/en-us/categories/360000539793-Release-Updates).
+
+# Install Agents
+This README.md file provides installation instructions for installing the agent from this Lacework Agent GitHub repository.
+You can also install the agent from the Lacework Console. For a listing of the different agent install methods, see [Agent Install Options](https://support.lacework.com/hc/en-us/sections/360002375733).
+
+## Install Prerequisites
+Complete the following prerequisites steps before starting any of the agent install procedures below.
+1. Verify that the Lacework agent supports the distribution installed your machine. For more information, see [Supported Operating Systems](https://support.lacework.com/hc/en-us/articles/360005230014).
+2. Use sed (GNU sed) 4.2.2 in the procedures below. 
```

Comment:
Would it better to include "or higher version". 

Basically I have tested the instructions to work with that version is assume newer ones would still work.

Reply:
I changed it to 'Use sed (GNU sed) version 4.2.2 or higher in the procedures below.'